src/Card/Card.stories.js

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import { Card } from "./Card";
+import { cardAlignOptions } from "./Card.levers";
+import { Heading } from "../Heading/Heading";
+import { CtaLink } from "../CtaLink/CtaLink";
+
+export default {
+  title: "Composite/Card",
+  component: Card,
+  argTypes: {
+    align: {
+      control: {
+        type: "inline-radio",
+        options: cardAlignOptions,
+      },
+    },
+    isMinimal: {
+      control: {
+        type: "boolean",
+      },
+    },
+  },
+};
+
+const CardTemplate = ({ ...rest }) => (
+  <Card {...rest}>
+    {/* Developer would pass in their own card content components */}
+    <span>Super headline</span>
+    <Heading>Headline</Heading>
+    <p>
+      Fusce convallis metus id felis luctus adipiscing. Vestibulum suscipit
+      nulla quis orci. Phasellus a est.
+    </p>
+    <CtaLink text="Call to action link" />
+  </Card>
```

Comment:
Please feel free to pass some props and attributes to these example children elements for a good "default" card, e.g., the headline level would most likely be an h3, and the size of the heading might be a 2 or 3. The body text could use the class `su-card-paragraph`. It would be good to also add `href` to the CTAs so the example will have a focus state and the right cursor 😃

Reply:
@rebeccahongsf Apologize I have a small correction to my comment - I think the size of the heading is more like 1 or 2 (I forget that 1 is already a step up from normal base font size (normal = 0, 1 = 1.25 x)) 😂

Reply:
And this is looking better and better - nice work 🥇 ! Let us know when you're ready for a real review so you have time to tweak and merge before sprint close 😃